### PR TITLE
Feature/autumn

### PR DESCRIPTION
**Pull request by jwtodd**
u/g to autumn latest

note: defer until we get a green travis build

### CODE REVIEW

File: modules/assignment/src/main/java/com/intuit/wasabi/assignment/impl/AssignmentsImpl.java [hunk 1]
```diff
@@ -67,16 +68,11 @@
  */
 public class AssignmentsImpl implements Assignments {
 
+    protected static final String RULE_CACHE = "ruleCache";
```
Comment (jwtodd) [inline modules/assignment/src/main/java/com/intuit/wasabi/assignment/impl/AssignmentsImpl.java]: @jcwuzoegiver  well as a part of u/g autumn i did not expect to see the included AssignmentsImpl mods ... wonder if i picked them up from somewhere else?

File: ATTRIBUTIONS/wasabi-api-attribution.xml [hunk 2]
```diff
@@ -1282,7 +1282,7 @@
          <type>jar</type>
          <licenses/>
          <downloadUrls>
-            <string>TBD</string>
+            <string>file:/Users/james/projects/wasabi/modules/api/../../repository/Hyrule/Hyrule/0.0.1/Hyrule-0.0.1.jar</string>
```
Comment (thgreiner) [inline ATTRIBUTIONS/wasabi-api-attribution.xml]: Should this really be in there?

File: ATTRIBUTIONS/wasabi-assignment-attribution.xml [hunk 3]
```diff
@@ -1141,7 +1141,7 @@
          <type>jar</type>
          <licenses/>
          <downloadUrls>
-            <string>TBD</string>
+            <string>file:/Users/james/projects/wasabi/modules/assignment/../../repository/Hyrule/Hyrule/0.0.1/Hyrule-0.0.1.jar</string>
```
Comment (thgreiner) [inline ATTRIBUTIONS/wasabi-assignment-attribution.xml]: Should this really be in there?

File: ATTRIBUTIONS/wasabi-user-directory-attribution.xml [hunk 4]
```diff
@@ -818,7 +818,7 @@
          <type>jar</type>
          <licenses/>
          <downloadUrls>
-            <string>TBD</string>
+            <string>file:/Users/james/projects/wasabi/modules/user-directory/../../repository/Hyrule/Hyrule/0.0.1/Hyrule-0.0.1.jar</string>
```
Comment (thgreiner) [inline ATTRIBUTIONS/wasabi-user-directory-attribution.xml]: Should this really be in here?

File: ATTRIBUTIONS/wasabi-main-attribution.xml [hunk 5]
```diff
@@ -1429,7 +1429,7 @@
          <type>jar</type>
          <licenses/>
          <downloadUrls>
-            <string>TBD</string>
+            <string>file:/Users/james/projects/wasabi/modules/main/../../repository/Hyrule/Hyrule/0.0.1/Hyrule-0.0.1.jar</string>
```
Comment (thgreiner) [inline ATTRIBUTIONS/wasabi-main-attribution.xml]: Should this really be in here?

File: ATTRIBUTIONS/wasabi-eventlog-attribution.xml [hunk 6]
```diff
@@ -807,7 +807,7 @@
          <type>jar</type>
          <licenses/>
          <downloadUrls>
-            <string>TBD</string>
+            <string>file:/Users/james/projects/wasabi/modules/eventlog/../../repository/Hyrule/Hyrule/0.0.1/Hyrule-0.0.1.jar</string>
```
Comment (thgreiner) [inline ATTRIBUTIONS/wasabi-eventlog-attribution.xml]: Should this really be in here?

File: ATTRIBUTIONS/wasabi-database-attribution.xml [hunk 7]
```diff
@@ -824,7 +824,7 @@
          <type>jar</type>
          <licenses/>
          <downloadUrls>
-            <string>TBD</string>
+            <string>file:/Users/james/projects/wasabi/modules/database/../../repository/Hyrule/Hyrule/0.0.1/Hyrule-0.0.1.jar</string>
```
Comment (thgreiner) [inline ATTRIBUTIONS/wasabi-database-attribution.xml]: Should this really be in here?

File: ATTRIBUTIONS/wasabi-api-attribution.xml (hunk 2 above)
Comment (jwtodd) [inline ATTRIBUTIONS/wasabi-api-attribution.xml]: nope. we have a post package/build script to sed out this string:

  see [ws]/bin/ATTRIBUTIONS_patch.sh

note: looking into using the mojo:license plugin.

thoughts?

File: ATTRIBUTIONS/wasabi-assignment-attribution.xml (hunk 3 above)
Comment (jwtodd) [inline ATTRIBUTIONS/wasabi-assignment-attribution.xml]: see above

File: ATTRIBUTIONS/wasabi-user-directory-attribution.xml (hunk 4 above)
Comment (jwtodd) [inline ATTRIBUTIONS/wasabi-user-directory-attribution.xml]: see above

File: ATTRIBUTIONS/wasabi-main-attribution.xml (hunk 5 above)
Comment (jwtodd) [inline ATTRIBUTIONS/wasabi-main-attribution.xml]: see above

File: ATTRIBUTIONS/wasabi-eventlog-attribution.xml (hunk 6 above)
Comment (jwtodd) [inline ATTRIBUTIONS/wasabi-eventlog-attribution.xml]: see above

File: ATTRIBUTIONS/wasabi-database-attribution.xml (hunk 7 above)
Comment (jwtodd) [inline ATTRIBUTIONS/wasabi-database-attribution.xml]: see above

**Comment by jwtodd** [comment]
@thgreiner good observation. we have a post-process filter to remove this (see comment). new patch forthcoming.

**Comment by jwtodd** [comment]
@thgreiner attribution filter applied, patch pushed

@iizrailevsky your eyes appreciated :)

@jcwuzoegiver thx re the AssignmentImpl changset ... i was just surprised to see that delta show up here.

squirrel me yo :)

**Comment by iizrailevsky** [comment]
@jwtodd 👍 on autumn package and GAV changes! :shipit: given that you resolve the DE code changes question

**Comment by wasabi-honeybadger** [comment]
ok build failed ... looking into that now

**Pull request closed without merging**